pkg/config/user_config.go

```diff
@@ -32,7 +32,7 @@ type GuiConfig struct {
 	ScrollHeight                int                `yaml:"scrollHeight"`
 	ScrollPastBottom            bool               `yaml:"scrollPastBottom"`
 	MouseEvents                 bool               `yaml:"mouseEvents"`
-	SkipUnstageLineWarning      bool               `yaml:"skipUnstageLineWarning"`
+	SkipDiscardChangeWarning    bool               `yaml:"skipDiscardChangeWarning"`
```

Comment:
This is a breaking change, but I didn't bother doing something about it. Since the warning prompt mentions the config, it's easy enough to fix it manually.

We could however consider rewriting the config file with the renamed property, now that that's possible. Worth it?

Reply:
I think it's worth rewriting the config. Note that the existing code in `pkg/utils/yaml_utils/yaml_utils.go` is not exactly what we need here: we need to replace a key rather than replace a value, but I assume it's similar enough logic